README.md

```diff
@@ -1,10 +1,15 @@
-# AxolotlClient-mod
+# Axolotl Client
 
-A Complete (nearly) mod to have various features on various Minecraft versions
+This README is also available in [Türkçe](https://github.com/AxolotlClient/AxolotlClient-mod/blob/multiversion/doc/README-tr.md) and [Deutsch](https://github.com/AxolotlClient/AxolotlClient-mod/blob/multiversion/doc/README-de.md).
 
-*Also available in other Languages: [Türkçe](doc/README-tr.md), [Deutsch](doc/README-de.md)*
+## Introduction
 
-### Features
+Axolotl  is a Client for Minecraft that introduces a range of features across various game versions, promoting a better, customizable environment for the user. 
```

Comment:
promoting?

Reply:
yea, just word choice, it could always be changed to something like: 'Creating a more flexible and customizable environment for the user.'

Reply:
yeah, that sounds more natural

Reply:
Just let me know if you want that to be revised!

Reply:
maybe `creating` would be better in this case